### PR TITLE
ApigeeX code to give option to enable add credit module during installation

### PR DESCRIPTION
**Pull request by divya-intelli**
This PR will show the monetization step along with option to enable add credit module during installation for monetized ApigeeX org.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -54,6 +54,9 @@
       },
       "drupal/radix": {
         "Fix issue to support upgrade of components to 2.x": "https://www.drupal.org/files/issues/2021-01-20/components-API-3193046-09.patch"
+      },
+      "apigee/apigee-client-php": {
+        "ApigeeX supported currency": "https://patch-diff.githubusercontent.com/raw/divya-intelli/apigee-client-php/pull/4.diff"
```
Comment (giteshk) [inline composer.json:59]: Why are we not merging this into the main library?

**Review by giteshk — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (divya-intelli) [reply composer.json:59]: Once we get a go ahead , feature-ngmint-phase2 branch will be merged to master branch, till then the PR's will be merged to feature-ngmint-phase2 branch.

**Review by raakesh-blokhra — APPROVED** [review]
LGTM

**Review by raakesh-blokhra — APPROVED** [review]
LGTM

**Review by raakesh-blokhra — APPROVED** [review]
LGTM

**Pull request merged**